### ISSUE TITLE
svviz demo can't parse answer to download prompt

### ISSUE THREAD

**Issue by 0xaf1f** [svviz/svviz#51]
With svviz 1.6.1 freshly installed via pip (python 2.7), I have issues running the `demo` subcommand:

```
$ svviz demo
Couldn't find example data in current working directory (svviz-examples/example1). Shall I download it and decompress it into the current working directory? Y/n:
Traceback (most recent call last):
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/bin/svviz", line 11, in <module>
    sys.exit(main())
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/app.py", line 371, in main
    run(sys.argv)
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/app.py", line 285, in run
    args = commandline.parseArgs(args)
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/commandline.py", line 104, in parseArgs
    inputArgs = checkDemoMode(args)
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/commandline.py", line 84, in checkDemoMode
    cmd = demo.loadDemo(which, args.auto_download)
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/demo.py", line 78, in loadDemo
    if not checkForDemo(which, autoDownload):
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/demo.py", line 68, in checkForDemo
    """Shall I download it and decompress it into the current working directory? Y/n:""".format(which))
  File "<string>", line 0
    
    ^
SyntaxError: unexpected EOF while parsing
```

if I try explicitly answering "Y" instead of just pressing enter:

```
$ svviz demo
Couldn't find example data in current working directory (svviz-examples/example1). Shall I download it and decompress it into the current working directory? Y/n:Y
Traceback (most recent call last):
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/bin/svviz", line 11, in <module>
    sys.exit(main())
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/app.py", line 371, in main
    run(sys.argv)
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/app.py", line 285, in run
    args = commandline.parseArgs(args)
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/commandline.py", line 104, in parseArgs
    inputArgs = checkDemoMode(args)
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/commandline.py", line 84, in checkDemoMode
    cmd = demo.loadDemo(which, args.auto_download)
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/demo.py", line 78, in loadDemo
    if not checkForDemo(which, autoDownload):
  File "/usr/local/Anaconda/envs_app/svviz/1.6.1/lib/python2.7/site-packages/svviz/demo.py", line 68, in checkForDemo
    """Shall I download it and decompress it into the current working directory? Y/n:""".format(which))
  File "<string>", line 1, in <module>
NameError: name 'Y' is not defined
```

**Comment by nspies** [comment]
Sorry about that, this issue was fixed in source (#49) but not released.

I have now cut a new release, 1.6.2, which now includes the fix. You can get the new release by updating from pypi: `pip install -U svviz`. Please let me know if you have any additional issues.

**Comment by 0xaf1f** [comment]
Oh, I missed that previous report since I only looked through open issues before filing this one. Thanks for making the new release.
